winter-website.css

```diff
@@ -1,4 +1,70 @@
 body {
+	padding: 0px;
+	margin: 0px;
 	color: #e8f3fc;
 	background-color: black;
+	font-family: sans-serif;
+}
+
+*, *::before, *::after {
+   box-sizing: border-box;
```

Comment:
you could change the fontsize to 10px here and use rem for rest of your sizing (margins, padding etc)

Reply:
rayna advised to only use rem for relative sizing regarding the font sizes. everything else should use % for relative sizing instead of rem